library/Fission/Web/Error.hs

```diff
@@ -58,6 +62,40 @@ instance ToServerError Int where
 
     n -> error <| show n <> " is not an error status code"
 
+instance ToServerError Error where
+  toServerError = \case
+    AddErr           addErr -> toServerError addErr
+    GetErr           getErr -> toServerError getErr
+    LinearizationErr linErr -> toServerError linErr
+
+instance ToServerError Get.Error where
+  toServerError = \case
+    Get.InvalidCID txt          -> err422 { errBody = UTF8.textToLazyBS txt }
+    Get.UnexpectedOutput _ -> err500 { errBody = "Unexpected IPFS result" }
+    Get.UnknownErr _         -> err500 { errBody = "Unknown IPFS error" }
+    (Get.TimedOut (CID hash) _) ->
```

Comment:
Does this need the outer brackets?

Reply:
nope that was an artifact from.... something